### PR TITLE
Fix alignment of nav contents

### PR DESCRIPTION
**Pull request by pulsejet**
Padding was only applicable to the list, but should apply to the footer too.

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
footer-before | footer_after

### CODE REVIEW

**Comment by pulsejet** [comment]
This doesn't play nice with scrollbars ...

**Review by Pytal — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Comment by pulsejet** [comment]
What's the general policy on merging after approval? Is it okay to merge to master after 2 approvals or wait for the maintainer to do it?

File: src/components/NcAppNavigation/NcAppNavigation.vue
Comment (raimund-schluessler) [inline src/components/NcAppNavigation/NcAppNavigation.vue:177]: @pulsejet Are you sure we can remove this rule? Maybe we should keep it, just in case? :thinking:

**Review by raimund-schluessler — APPROVED** [review]

**Pull request merged**
